ui/app/(prowler)/lighthouse/config/page.tsx

```diff
@@ -0,0 +1,274 @@
+"use client";
```

Comment:
Please don't add use client to any page in the app. We're using SSR capabilities from Next.js 14. There are no pages with use client in the entire project, so let's avoid introducing it.

For references: https://nextjs.org/docs/14/app/building-your-application/rendering

Reply:
I tried to not use client, but somehow because of AI SDK, use client is the only solution I found.

If I move all logic from chat.tsx to page.tsx and remove "use client", the page errors out with the following:
```
 ⨯ ./node_modules/@ai-sdk/react/dist/index.mjs
Attempted import error: 'swr' does not contain a default export (imported as 'useSWR').

Import trace for requested module:
./node_modules/@ai-sdk/react/dist/index.mjs
./app/(prowler)/lighthouse/page.tsx
```

I don't know how to solve this. Please let me know if there's a way. Thanks!

Reply:
Hi, I dont know how to fix this. If I remove use client, the configured API keys and business context are not shown. Please let me know how to fix this.